core/src/main/java/jenkins/util/VirtualFile.java

```diff
@@ -148,7 +148,7 @@
      * Some implementations may not support symlinks under any conditions.
      * @return a target (typically a relative path in some format), or null if this is not a link
      * @throws IOException if reading the link, or even determining whether this file is a link, failed
-     * @since FIXME
+     * @since 2.118
```

Comment:
Not sure about that. APIs are in the Beta status. Is annotation enough for Javadoc in this case? CC @jglick